frontend/src/pages/docs/[...slug].tsx

```diff
@@ -333,13 +337,13 @@ const DocContent = styled.div`
   }
 `;
 
-const StyledImage = styled(Image)``;
+const StyledImage = styled(Image)();
 
 const ImageContainer = styled.div`
   width: 100%;
   margin: 24px 0;
 
-  > div {
+  > span {
```

Comment:
this change is needed since Next12 changed the Image wrapper from `div` to `span`:

https://nextjs.org/docs/upgrading#nextimage-changed-wrapping-element

Reply:
Images look good now! Thanks so much for catching this, @seve 🙏 

<img width="1792" alt="Screen Shot 2022-08-24 at 10 41 49 AM" src="https://user-images.githubusercontent.com/6309723/186487514-68a9d455-d64f-4678-9359-c49129cb7900.png">